.github/workflows/qaqc_deploy.yml

```diff
@@ -49,7 +49,7 @@ jobs:
         run: |
           ssh qaqc "\
             docker pull $DOCKER_IMAGE_NAME
-            pid=\"\$(docker ps -a -q -f ancestor=$DOCKER_IMAGE_NAME)\"
+            pid=\"\$(docker ps -a -q -f)\"
```

Comment:
-f needs to be dropped if no filter is being used.

Reply:
d'oh. Yes. Fixed @fvankrieken